docs/pages/tutorial/gestures.mdx

```diff
@@ -144,7 +144,7 @@ const imageStyle = useAnimatedStyle(() => {
     width: withSpring(scaleImage.value),
     height: withSpring(scaleImage.value),
   };
-});
+}, [scaleImage]);
```

Comment:
This seemed like the right dependency to add because it is the value this style depends on/modifies.